locales/en.default.schema.json

```diff
@@ -491,6 +491,9 @@
         },
         "show_vendor": {
           "label": "Show vendor"
+        },
+        "show_rating": {
+          "label": "Show product rating"
```

Comment:
Thank you, @melissaperreault ! Looking at that version along with the other settings, I can refine it to be singular and more like metafield support copy:
`To display a product rating, add a product ratings app. Learn more [offsite icon]`

If we say `product rating` too much since this is under `Show product rating`
`To display a rating, add a product ratings app. Learn more [offsite icon]`

In Sofia's [prototype video](https://screenshot.click/16-41-o5hev-b3q21.mp4), the block is called `Rating`; would adjust that to `Product rating`

Reply:
Thank you Gretchen! 
I am aligned with this proposition, looks good to me! `To display a rating, add a product ratings app. Learn more [offsite icon]`

I made the edit in the suggestions too!

Reply:
@gretchen-willenberg I was wondering if it should align with the metafield names, if not, aligned with matching merchants vocabulary that they are used to! Finally, the docs team can adapt with our preference!